tutorials/packaging/snap-a-website.md

```diff
@@ -0,0 +1,670 @@
+    install: |
+      mkdir $SNAPCRAFT_PART_INSTALL/app
+      mv dist/linux-unpacked/* $SNAPCRAFT_PART_INSTALL/app
+    prime:
+      - -node_modules
+```
+
+### Let's recap
+
+We have added general metadata to ensure our app appears correctly in stores and parts to ensure it bundles our executable and its dependencies in a confined package. But we are still missing one very important bit: the launcher.
+
+## Launcher and desktop integration
+
+To launch the app, the snap needs to know which command to run. This is done within an `apps` section in the `snapcraft.yaml`, where we are going to declare: the name of the app, the command to run and which access we want the app to be granted outside of the snap confinement.
+
+Positive
+: **About confinement**
+By default, a snap will be fully confined and will have no way to interact with other pieces of the system, such as accessing the network, the display and sound servers, etc. As a result, we need to declare explicitly what our app needs access to.
+
+This is what our `apps` section needs to look like.
+
+```yaml
+apps:
+  castlearena:
+    command: env TMPDIR=$XDG_RUNTIME_DIR desktop-launch $SNAP/app/castlearena
+    plugs:
+      - home
+      - x11
+      - unity7
+      - browser-support
+      - network
+      - gsettings
+      - pulseaudio
+      - opengl
+```
+
+This list of "`plugs`", which are permissions that connects into similarly named "`slots`" on the user system, is more or less what any desktop application would need.
+
+### All the pieces together
+
+By adding the `apps` section to your `snapcraft.yaml`, you should now have a file looking like this:
+
+```yaml
+name: castlearena
+version: '0.1'
+summary: Destroy your opponent's castle to win!
+description: |
+ Play online or against a bot, click and drag cards to your side of the field
+ to deploy powerful building and units to attack your ennemy.
+
+confinement: strict
+grade: stable
+
+parts:
+  electron-app:
+    plugin: nodejs
+    source: app/
+
+    stage-packages:
+      - libnotify4
+      - libappindicator1
+      - libxtst6
+      - libnss3
+      - libxss1
+      - fontconfig-config
+      - gconf2
+      - libasound2
+      - pulseaudio
+
+    after:
+      - desktop-glib-only
+
+    build: node_modules/.bin/electron-builder
+    install: |
+      mkdir $SNAPCRAFT_PART_INSTALL/app
+      mv dist/linux-unpacked/* $SNAPCRAFT_PART_INSTALL/app
+    prime:
+      - -node_modules
+
+apps:
+  castlearena:
+    command: env TMPDIR=$XDG_RUNTIME_DIR desktop-launch $SNAP/app/castlearena
+    plugs:
+      - home
+      - x11
+      - unity7
+      - browser-support
+      - network
+      - gsettings
+      - pulseaudio
+      - opengl
+```
+
+### Icon and desktop file
+
+The last bit of packaging we need is an icon and a `<app>.desktop` file so desktop environments recognize our app as such. The icon and the desktop file will be picked up by snapcraft and handled accordingly by putting them in a `snap/gui/` directory.
```

Comment:
you can either have "so desktop environments **will**" or "so **that** desktop environments..."